NG-BSCSD-04-01/myproject/src/main/java/newpackage/Student.java

```diff
@@ -0,0 +1,27 @@
+
+package newpackage;
+
+/**
+ *
+ * @author PC
+ */
+public class Student {
+      
+    
+    public String firstName ;
```

Comment:
@maiya20 Please move these attributes to Person class

Reply:
Ok